Assets/VRTK/Source/Scripts/Locomotion/ObjectControlActions/VRTK_BaseRotateObjectControlAction.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Base Rotate Object Control Action|ObjectControlActions|25010
+namespace VRTK
+{
+    using UnityEngine;
+
+    /// <summary>
+    /// Provides a base that all rotate object control actions can inherit from.
+    /// </summary>
+    /// <remarks>
+    /// **Script Usage:**
+    ///   > This is an abstract class that is to be inherited to a concrete class that provides rotate object control action functionality, therefore this script should not be directly used.
+    ///   > If a new rotate object control action scripts inherits from this script and it is possible that the controlled game object is the player, use the methods 'CheckForPlayerBeforeRotation' before and 'CheckForPlayerAfterRotation' after the rotation to compensate player's head offset due to rotation.
+    /// </remarks>
+    public abstract class VRTK_BaseRotateObjectControlAction : VRTK_BaseObjectControlAction
```

Comment:
seems a bit overkill creating a whole new abstracted class for these two methods.

I'd recommend either just sticking them in the `VRTK_BaseObjectControlAction` so there's no need for this class, or creating a class that can be used via composition rather than having triple inheritance.